astro/src/content/docs/get-started/use-cases/b2b2c.mdx

```diff
@@ -0,0 +1,190 @@
+
+### Implementation Notes
+
+Each customer's users are entirely separate with this tenant based implementation. A person can sign up with the email address `jamie.morgan@example.com` for both Ringling Brothers and Cirque De Soleil. The person who owns the `jamie.morgan@example.com` email address may not even know the ticketing application for these circuses are both provided by ClownPass.
+
+Each account is entirely separate with no relation to each other. The accounts can have different:
+
+* profile attributes
+* credentials
+* roles in applications
+
+### Alternative Implementation
+
+The tenant based solution outlined above has distinct user spaces. If you want a shared user space, where the `jamie.morgan@example.com` user has the same profile data and credentials no matter which circus they are buying tickets for, you need to use a different approach. Instead of separate tenants, have all users in one tenant and model each circus ticketing application as a separate FusionAuth Application configuration. Make sure to build and enforce your own authorization scheme to ensure customer admins from Ringling Brothers couldn't access Cirque De Soleils' users, since you can't use tenant locked API keys.
+
+Here's a table which shows tradeoffs between these different approaches.
+
+<UserSegmentTable />
+
+You might want to use one of the alternative options if users must be able to seamlessly purchase tickets from different circuses with a single account. This is similar to the approach of GitHub or Slack.
+
+## Expected Outcome
+
+Your web or mobile application has isolated groups of users, one set for each customer. Each user has their own profile data and manages their identity, either in your application or via other login methods such as social sign-on.
+
+Admins can use whatever user management related functionality you choose to expose in your application, using tenant locked API keys to limit access.
+
+## Edge Cases
+
+The word `tenant` can be overloaded. When you are planning for this use case, make sure you distinguish between tenants in your application and FusionAuth tenants. They don't have to map one to one.
+
+<CrossTenantUserAccess />
+
+## Other Example Scenarios
+
+Any application which has customers who then have their own users or customers is a good fit for this use case.
+
+A very common scenario is a private labelled product sold to SMBs or consumers. Examples include:
+
+* Website builders resold by creative agencies
+* Appointment schedulers for hair salons, personal trainers and other service providers
+* Private labelled job boards
+* Accounting software sold to CPAs and used by the customers they offer tax services to
+
+In each of these cases there is a clear differentiator between the customer organization buying the solution and their end customer. This use case is appropriate when the identity of the end user is *not* controlled by the customer, but rather by the end user themselves.
```

Comment:
Should 'differentiator' be 'differentiation' instead? Singular, with 'differentiator,' would imply there is one specific things that seperates the customer's and their users.

Reply:
good point.